metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/query/request/SearchFieldConfig.java

```diff
@@ -21,7 +21,7 @@
 public class SearchFieldConfig {
     public static final float DEFAULT_BOOST = 1.0f;
 
-    public static final Set<String> KEYWORD_FIELDS = Set.of("urn", "runId");
+    public static final Set<String> KEYWORD_FIELDS = Set.of("urn", "runId", "_index");
```

Comment:
any way we could have caught this with one of your unit tests?

Reply:
This is downstream of that, let me see if I can find a part where this can be tested...